stories/topics/proc-aap-gcp-backup-and-restore.adoc

```diff
@@ -0,0 +1,176 @@
+=====
++
+[source,bash]
+----
+$ export IMAGE=quay.io/ansible-on-clouds/ansible-on-clouds-ops:2.2.1
+$ docker pull $IMAGE
+----
++
+. Running the backup playbook as a container.
++
+.. Replace `<absolute path to google application credentials dir>` with your dir path to the service account credential Json file for google cloud and `<absolute path of extra vars dir>` with the path to your directory where you `extra_vars.yaml` file exists which you have created in step 1 of backup process. 
++
+.. Use the following command to run the playbook.
++
+[source,bash]
+----
+$ docker run --rm \
+-v <absolute path to google application credentials dir>:/secrets:ro \
+-v <absolute path of extra vars dir>:/extra_vars:ro \
+$IMAGE \
+redhat.ansible_on_clouds.gcp_backup_deployment \
+-e @/extra_vars/extra_vars.yaml
+----
++
+. Note the output of the playbook.
++
+The playbook will return something like this
++
+[source, bash]
+----
+TASK [redhat.ansible_on_clouds.standalone_gcp_backup : [backup_deployment] Print vars required for restore process] ***
+ok: [localhost] => {
+    "msg": [
+        "AAP on GCP Backup successful. Please note below vars which are required for restore process.",
+        "{ gcp_sql_backup_id: 1677552759614 , gcp_sql_backup_db_name: test-bnr-aap-db ,gcp_filestore_backup_name: test-bnr-filestore-iygs }"
+    ]
+}
+----
++
+Make a note of the SQL database backup ID, SQL batabase backup name and the filestore backup name. These will be needed for the restore playbook.
++
+
+
+
+== Restore Process
+
+The restore process deploys a new deployment, and restores the filestore and SQL database instance to the specified backup.
+
+[NOTE]
+=====
+The restored filestore instance uses CIDR network range `192.168.244.0/29`, which is different from the default network filestore network range of the backup deployment.
+=====
+
+The following instructions describe how to run the restore playbook.
+
+.Procedure
+. Gather the following input variables to create an `extra_vars.yaml` file.
+.. *gcp_service_account_credentials_json_path*: Service account crendential json file path for the google cloud account. This is the path within "ansible-on-clouds-ops" container where credentials json file is mounted to `/secrets` dir. The path should be `/secrets/<service-account-creds-json-filename>`, Replace `<service-account-creds-json-filename>` with your google service account credential filename.
++
+[NOTE]
+=====
+To create the GCP credentials file, see, 
+link:https://developers.google.com/workspace/guides/create-credentials[Create credentials] and 
+link:https://cloud.google.com/iam/docs/keys-create-delete[Create and delete service account keys]
+=====
++
+.. *gcp_deployment_name*: Deployment name which you backed up.
+.. *gcp_restored_deployment_name*: Deployment name of restored AAP foundation created by playbook
```

Comment:
@leena-jawale : this cannot be same as any deployment that is up and running already as noted by you on slack

kindly revise the same